DQM/EcalMonitorClient/interface/TimingClient.h

```diff
@@ -15,7 +15,8 @@ namespace ecaldqm {
   private:
     void setParams(edm::ParameterSet const&) override;
 
-    float toleranceMean_;
+    float EBtoleranceMean_;
```

Comment:
Please modify the class member with lowercase initial, e.g. ebToleranceMean_ and eeToleranceMean_

Reply:
Sure.